app/assets/javascripts/test/backend-snapshot-tests/teamstructure.e2e.js

```diff
@@ -0,0 +1,170 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
+/* eslint-disable import/first */
+// @flow
+import {
+  tokenUserDefault,
+  tokenUserAAB,
+  tokenUserAAC,
+  tokenUserABA,
+  tokenUserBAA,
+  setCurrToken,
+} from "../enzyme/e2e-setup";
+import test from "ava";
+import _ from "lodash";
+import * as api from "admin/admin_rest_api";
+
+/*
+TEAM STRUCTURE USED FOR TESTING:
+
+Organizations:
+Connectomics department (Organization A), OrganizationB
+
+Teams:
+Connectomics department (Team AA), Team AB, Team BA, test1, test2
+(the teams test1 and test2 existed before)
```

Comment:
The comment "...existed before" only makes sense in the context of this PR. After merging it will not be clear what "before" refers to :)
Are test1 and test2 still necessary?